Forms/main.css

```diff
@@ -1,57 +1,84 @@
 body{
-    width: 1800px;
-    height: 900px;
-    /*background-color: yellow;*/
+    width: 100vw;
+    height: 100vh;
 }
 
 div.container{
     display: flex;
-    margin: 10px    ;
+    margin: 10px;
+    justify-content: center;
+}
+div.head-container{
+    display: flex;
+    margin: 10px;
+    flex-direction: row-reverse;
+    justify-content: center;
 }
-
 h1{
     color: darkgray;
-    width: 95%;
+    width: 100%;
     height: 100px;
     text-align: center;
+    padding-right: 200px;
 }
 
 footer{
-    position: relative;
-    width: 95%;
-    height: 100px;
+    background-color: aquamarine;
+    width: 100% ;
+    height: 50px;
     text-align: center;
     margin: 40px 0 0 40px;
 }
 
 .img1 {
-    position: absolute;
+    position: relative;
+    display: flex;
     width: 200px;
     height: 150px;
-    margin: 10px;
-
+    margin-left: 40px;
 }
 
-div.first_collum{
-
+div.first_column{
+    width: 200px;
+    /*margin-left: 60px;*/
+    margin-right: 60px;
     display: flex;
-    justify-content: space-between;
-    flex-direction: column;
+    justify-content: center;
 }
 
-div.secound_collum{
-    width: 700px;
-    margin-left: 60px;
-    margin-right: 60px;
+div.second_column{
     display: flex;
     justify-content: center;
+    flex-direction: column;
+    flex: 2;
 }
 
 .text_form{
     display: flex;
     justify-content: space-between;
     margin: 0px 10px 5px;
-    padding-right: 300px;
+    padding:0 100px;
+}
+
+div.form{
+    display:flex;
+    justify-content: end;
+    flex-direction: column;
+    text-align: center;
```

Comment:
You can set `justify-content: center;` to align the content to center. Try it out

Reply:
I try but items no change, only text-align can replase position in center

Reply:
hm, interesting. we'll look into it surely